src/yoti_common/attribute.converter.js

```diff
@@ -21,6 +25,13 @@ module.exports.AttributeConverter = class AttributeConverter {
     switch (attrName) {
       case constants.ATTR_DOCUMENT_DETAILS:
         return new DocumentDetails(value);
+      case constants.ATTR_DOCUMENT_IMAGES:
+        if (!(value instanceof MultiValue)) {
```

Comment:
I think this bit would want to fail

Reply:
Now throwing error in d6edd2d, which will be caught and logged